test/test_normalized.jl

```diff
@@ -3,14 +3,16 @@ import ClassicalOrthogonalPolynomials: NormalizationConstant, NormalizedBasisLay
 import LazyArrays: CachedVector, PaddedLayout
 import ContinuumArrays: MappedWeightedBasisLayout
 
+
+
 @testset "Normalized" begin
     @testset "Legendre" begin
         P = Legendre()
         Q = Normalized(P)
 
         @testset "Basic" begin
             @test MemoryLayout(Q) isa NormalizedBasisLayout
-            @test @inferred(Q\Q) ≡ Eye(∞)
+            @test_broken @inferred(Q\Q) ≡ Eye(∞)
```

Comment:
This line throws an error for me. I'll have to check if this is somehow caused on my end or if this is happening on master branch too. Everything else seems fine with this.

Reply:
What's the error?

Reply:
I'm getting

```julia
julia> P = Legendre()
Legendre{Float64}

julia> Q = Normalized(P)
Normalized(Legendre{Float64})

julia> Q\Q
∞×∞ LinearAlgebra.Diagonal{Float64, Ones{Float64, 1, Tuple{InfiniteArrays.OneToInf{Int64}}}} with indices OneToInf()×OneToInf()

julia> Eye(∞)
∞×∞ LinearAlgebra.Diagonal{Float64, Ones{Float64, 1, Tuple{InfiniteArrays.OneToInf{Int64}}}} with indices OneToInf()×OneToInf()

julia> @inferred(Q\Q) ≡ Eye(∞)
ERROR: LoadError: return type LinearAlgebra.Diagonal{Float64, Ones{Float64, 1, Tuple{InfiniteArrays.OneToInf{Int64}}}} does not match inferred return type Any
Stacktrace:
 [1] error(s::String)
   @ Base ./error.jl:33
 [2] top-level scope
   @ ~/Documents/Projects/ClassicalOrthogonalPolynomials.jl/test/test_normalized.jl:11
```

Reply:
hmm that's also broken on mine...

Reply:
ohhh... is the error being thrown in the REPL or when running as a test? 

The REPL isn't as good for type inference... but we can always just remove the `@inferred`

Reply:
Yes, this was in REPL. In that case the normalized Legendre stuff based on explicit formulae seems to work fine then.